### DISCUSSION TITLE
CSS Breaking after v10.5.1

### DISCUSSION THREAD

**Discussion by jnlinn** [primefaces/community#1650] [PrimeReact] [answered]
We are currently using PrimeReact v10.5.1 with primeflex 3.3.1 and using Diamond template v6. Running React 17.

Trying to update the latest version of PrimeReact but getting issue with CSS as shown in images below. Any ideas how to fix?

Checkboxes are duplicated:
![checkbox-duplicated](https://github.com/primefaces/community/assets/88069194/70ab1f3a-d671-49e6-908d-797457a5ac60)

SelectButton inner corner radius not setting to 0. (same issue for multiselect dropdown and other components)
![selectbutton-css](https://github.com/primefaces/community/assets/88069194/2cdd8e5e-5567-4da1-8225-a488f3944c86)

Best regards

**Comment by melloware** [comment, marked as answer]
See: https://github.com/primefaces/primereact/issues/6328

you need to update your theme.css

**Reply by jnlinn** [reply]
Thank you for the quick response. I followed primereact#6328 and added the following to /sass/overrides/theme_style.scss to try and fix checkboxes:

.p-checkbox {
  position: relative;
  display: inline-flex;
  user-select: none;
  vertical-align: bottom;
}
.p-checkbox-input {
  appearance: none;
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  padding: 0;
  margin: 0;
  opacity: 0;
  z-index: 1;
  outline: 0 none;
  cursor: pointer;
}
.p-checkbox-box {
  display: flex;
  justify-content: center;
  align-items: center;
}

This is the result:

![check-boxes](https://github.com/primefaces/community/assets/88069194/95948e8a-6aaf-4022-b582-0a5f91448025)

"Primary" and "Billing" are checked.

It fixed the overlay issue, however the theme component background color is not being applied.

**Reply by melloware** [reply]
If you are using a theme you can copy the SCSS from here: https://github.com/primefaces/primereact-sass-theme

**Reply by jnlinn** [reply]
I am using the Diamond Template v6

**Reply by melloware** [reply]
Yep so am I. You can just replace the SCSS files in your Diamond Template and regenerate.  I just did it yesterday and my Diamond is working on 10.6.2 just fine.  All the files in `themes-base` you can just drop right into your Diamond Template `\sass\theme\designer` folder.

**Reply by jnlinn** [reply]
That's working! Why aren't the \sass\theme\designer files getting updated when I update PrimeReact?

**Reply by melloware** [reply]
That is in your premium theme so usually primetek only updates on major releases so Diamond 7.0 for PR 11.0 will have these changes but on the interim you typically just update.  I just watch the https://github.com/primefaces/primereact-sass-theme repo...
